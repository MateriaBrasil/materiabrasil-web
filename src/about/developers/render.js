import React from 'react'

import Colors from '../../Colors'
import Session from '../Session'
import Members from '../Members'
import Title from '../Title'
import Subtitle from '../Subtitle'
import Paragraph from '../Paragraph'
import developersList from './developersList'

export default () => (
  <Session id="developers">
    <Title color>Desenvolvedores</Title>
    <Paragraph color>
      Para o desenvolvimento desses direcionadores a MateriaMundi contou com uma
      equipe de consultores especialistas em diversas áreas.
    </Paragraph>
    <Subtitle>Conheça o nosso time!</Subtitle>
    <Members list={developersList} color={Colors.black} />
  </Session>
)
