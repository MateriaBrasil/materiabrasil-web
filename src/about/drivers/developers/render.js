import React, { Fragment } from 'react'

import Members from '../../Members'
import Subtitle from '../../Subtitle'
import Paragraph from '../../Paragraph'
import membersList from './developersList'

export default () => {
  return (
    <Fragment>
      <Subtitle>Desenvolvedores</Subtitle>
      <Paragraph>
        Para o desenvolvimento desses direcionadores a MateriaMundi contou com
        uma equipe de consultores especialistas em diversas áreas.
      </Paragraph>
      <Paragraph>Conheça o nosso time!</Paragraph>
      <Members membersList={membersList} />
    </Fragment>
  )
}
