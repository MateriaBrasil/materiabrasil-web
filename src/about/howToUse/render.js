import React from 'react'

import Section from '../Section'
import Title from '../Title'
import Video from '../../Video'
import VideoStyle from './VideoStyle'

export default () => (
  <Section
    id="how_to_use"
    backgroundImage="url('https://s3.amazonaws.com/materiamundi-us/static/backgrounds/material-3.jpg')"
  >
    <Title>Como usar</Title>
    <VideoStyle>
      <Video />
    </VideoStyle>
  </Section>
)
