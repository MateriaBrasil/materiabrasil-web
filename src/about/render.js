import React, { Fragment } from 'react'
import Typography from '@material-ui/core/Typography'

import Grid from './Grid'
import NavBar from '../navBar/NavBar'

export default props => routeProps => (
  <Fragment>
    <NavBar {...props} isTransparent="true" />

    <Grid>
      <Typography
        variant="display2"
        color="inherit"
        className="text"
        style={{
          color: '#fff',
          marginTop: 70,
          display: 'inline-block',
          fontWeight: 300,
        }}
      >
        <h2>
          <strong>Objetivo</strong>
        </h2>
        O objetivo da MateriaMundi oferecer acesso à informação de qualidade e
        parâmetros para que seus usuários possam escolher alternativas
        comercialmente disponíveis e socioambiental responsáveis para os
        desafios do dia a dia, variando desde o que vestimos até como
        construímos nossas casas.
        <br />
        Através da criação de uma comunidade ativa a plataforma permite a
        criação de conhecimento compartilhado sobre ações práticas para uma
        forma de projetar, produzir e consumir mais responsável com a sociedade
        e o meio ambiente.
        <h2>
          <strong>Como usar</strong>
        </h2>
        <h2>
          <strong>Equipe</strong>
        </h2>
        <h2>
          <strong>Board</strong>
        </h2>
        <h2>
          <strong>Direcionadores</strong>
        </h2>
      </Typography>
    </Grid>
  </Fragment>
)
