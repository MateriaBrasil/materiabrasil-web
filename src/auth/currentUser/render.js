import React from 'react'

import Loading from '../../Loading'

export default ({ render }) => (currentUser, props) => {
  const { fetchingInfo, info, infoError } = props

  if (fetchingInfo || (!info && !infoError)) {
    return <Loading />
  }

  return render(info, props)
}
