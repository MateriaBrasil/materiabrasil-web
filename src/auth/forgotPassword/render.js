import React from 'react'

import ForgotPassword from './ForgotPassword'

export default props => routeProps => (
  <ForgotPassword {...props} {...routeProps} />
)
