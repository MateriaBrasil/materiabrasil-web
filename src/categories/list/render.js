import React from 'react'

import Filters from 'filters/Filters'
import List from './List'

export default props => list => (
  <Filters
    render={filters => <List {...props} filters={filters} list={list} />}
  />
)
