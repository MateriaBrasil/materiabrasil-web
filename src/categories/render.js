import React from 'react'
import { List } from 'croods'

import Dialog from 'materials/Dialog'
import renderList from './renderList'

export default props => routeProps => (
  <Dialog {...props} {...routeProps} title="Editar informações técnicas">
    <List name="categories" render={renderList({ ...props, ...routeProps })} />
  </Dialog>
)
