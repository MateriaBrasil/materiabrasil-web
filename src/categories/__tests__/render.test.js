import React from 'react'
import renderer from 'react-test-renderer'

import render from '../render'

jest.mock('croods', () => ({
  List: props => <div {...props}>List -{props.render()}</div>,
}))

jest.mock('../renderList', () => props => list => (
  <div {...props} list={list}>
    renderList
  </div>
))

jest.mock('materials/Dialog', () => props => (
  <div {...props}>Dialog - {props.children}</div>
))

it('render correctly', () => {
  const routeProps = { match: { params: { id: '1234' } } }
  const tree = renderer.create(render({ bar: 'foo' })(routeProps)).toJSON()
  expect(tree).toMatchSnapshot()
})
