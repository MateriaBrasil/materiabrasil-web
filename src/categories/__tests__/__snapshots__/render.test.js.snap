// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  bar="foo"
  expandedCategories={Array []}
  match={
    Object {
      "params": Object {
        "id": "1234",
      },
    }
  }
  onChangeExpanded={[Function]}
  render={[Function]}
  title="Editar informações técnicas"
>
  Dialog - 
  <div
    name="categories"
    render={[Function]}
  >
    List -
    <div
      bar="foo"
      expandedCategories={Array []}
      list={undefined}
      match={
        Object {
          "params": Object {
            "id": "1234",
          },
        }
      }
      onChangeExpanded={[Function]}
      render={[Function]}
    >
      renderList
    </div>
  </div>
</div>
`;
